guide/src/module.md

```diff
@@ -1,44 +1,49 @@
 # Python Modules
 
-You can create a module as follows:
+You can create a module using `#[pymodule]`:
 
 ```rust
 use pyo3::prelude::*;
+use pyo3::wrap_pyfunction;
 
-// add bindings to the generated Python module
-// N.B: "rust2py" must be the name of the `.so` or `.pyd` file.
```

Comment:
This is said later in the paragraph. It's repetitive and distracting to say here too.